packages/react-composites/review/stable/react-composites.api.md

```diff
@@ -160,10 +169,18 @@ export interface CallAdapterCallOperations {
     startCamera(options?: VideoStreamOptions): Promise<void>;
     startCaptions(options?: StartCaptionsOptions): Promise<void>;
     startScreenShare(): Promise<void>;
+    // @beta
```

Comment:
These new properties should not have the beta annotation.

Reply:
Done